.github/workflows/prod-workflow.yaml

```
@@ -92,6 +92,51 @@ jobs:
           SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           NONCE_LAB_API_TOKEN: ${{ secrets.NONCE_LAB_API_TOKEN }}
+          INTERNAL_SERVER_KEY: ${{ secrets.VOICE_KOREA_INTERNAL_SERVER_KEY }}
         run: |
           npm i -g aws-cdk
           ENV=dev make deploy-api-if-needed
+        
+  watcher-deployment:
+    runs-on: ubuntu-22.04
+
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)